oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/record-requests/components/RequestModal.jsx

```diff
@@ -115,9 +118,17 @@ export const RequestModal = ({ request, requestTypes, requestModalType, isEventM
   }
 
   const createAndSubmitRequest = async () => {
+    let createPayload = formik.values;
+    if (isEmpty(formik.values?.payload?.content) && Object.keys(formik.values?.payload).length <= 1) {
+      createPayload = {};
```

Comment:
`content` is missing from default payload, but passed in when `formik.values` not empty